server/controllers/cohort.js

```diff
@@ -506,7 +512,7 @@ const leaveCohort = (req, resp) => {
 
       const { memberIds, ownerIds } = doc;
 
-      if (isOwner(doc, userId)) {
+      if (isCohortOwner(doc, userId)) {
```

Comment:
in line 186. you have this assigned to a variable and here you are calling this within if. 
Please select one approach to be consistent